package.json

```diff
@@ -46,6 +46,7 @@
     "@frontside/eslint-config": "^1.1.0"
   },
   "resolutions": {
+    "globby": "11.0.1",
```

Comment:
@pittst3r you are very right to ask.  I added it because it is getting clobbered by `webpack-dev-server`.  Here is what `yarn list globby` returns without the resolution

```
yarn list v1.22.4
warning Filtering by arguments is deprecated. Please use the pattern option instead.
├─ del@5.1.0
│  └─ globby@10.0.2
├─ globby@11.0.1
├─ react-dev-utils@10.2.1
│  └─ globby@8.0.2
└─ webpack-dev-server@3.10.3
   └─ globby@6.1.0
```

Reply:
Hmm, I just tried removing this and everything looks good according to the test suite. This is what I see when I run `yarn why globby`. What do you see?

```
$ yarn why globby
yarn why v1.22.4
[1/4] 🤔  Why do we have the module "globby"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "globby@11.0.1"
info Has been hoisted to "globby"
info Reasons this module exists
   - "workspace-aggregator-ca912f35-7901-4f63-a828-3e39a7375d87" depends on it
   - Hoisted from "_project_#@bigtest#server#globby"
   - Hoisted from "_project_#@changesets#cli#@manypkg#get-packages#globby"
info Disk size without dependencies: "688KB"
info Disk size with unique dependencies: "1.3MB"
info Disk size with transitive dependencies: "6.32MB"
info Number of shared dependencies: 36
=> Found "react-dev-utils#globby@8.0.2"
info This module exists because "_project_#@bigtest#todomvc#react-scripts#react-dev-utils" depends on it.
info Disk size without dependencies: "24KB"
info Disk size with unique dependencies: "740KB"
info Disk size with transitive dependencies: "5.96MB"
info Number of shared dependencies: 44
=> Found "del#globby@10.0.2"
info This module exists because "_project_#@bigtest#webdriver#chromedriver#del" depends on it.
info Disk size without dependencies: "40KB"
info Disk size with unique dependencies: "1.59MB"
info Disk size with transitive dependencies: "7.66MB"
info Number of shared dependencies: 46
=> Found "webpack-dev-server#globby@6.1.0"
info Reasons this module exists
   - "_project_#@bigtest#todomvc#react-scripts#webpack-dev-server#del" depends on it
   - Hoisted from "_project_#@bigtest#todomvc#react-scripts#webpack-dev-server#del#globby"
info Disk size without dependencies: "40KB"
info Disk size with unique dependencies: "172KB"
info Disk size with transitive dependencies: "440KB"
info Number of shared dependencies: 13
✨  Done in 0.98s.
```

According to:

```
=> Found "globby@11.0.1"
info Has been hoisted to "globby"
```

`globby@11.0.1` is getting hoisted to the root of the project while the other versions are getting installed within the `node_modules` of their respective packages, so it should all be fine. Is there another way I can reproduce the conflict you were seeing?

Reply:
you are right, totally not needed.

No idea why I added now.

Removed